integrations/templates/setup.md

```diff
@@ -0,0 +1,82 @@
+[% if entry.setup.prerequisites.list %]
+### Prerequisites
+
+[% for prereq in entry.setup.prerequisites.list %]
+#### [[ prereq.title ]]
+
+[[ prereq.description ]]
+
+[% endfor %]
+[% endif %]
+[% if entry.setup.configuration.file.name %]
+### Configuration
+
+#### File
+
+The configuration file name for this integration is `[[ entry.setup.configuration.file.name ]]`.
+[% if 'section_name' in entry.setup.configuration.file %]
+Configuration for this specific integration is located in the `[[ entry.setup.configuration.file.section_name ]]` section within that file.
+[% endif %]
+
+[#
+The file format is YAML. Generally, the format is:
+
+```yaml
+update_every: 1
+autodetection_retry: 0
+jobs:
+  - name: some_name1
+  - name: some_name1
+```
+#]
+
+You can edit the configuration file using the `edit-config` script from the
+Netdata [config directory](https://github.com/netdata/netdata/blob/master/docs/configure/nodes.md#the-netdata-config-directory).
+
+```bash
+cd /etc/netdata 2>/dev/null || cd /opt/netdata/etc/netdata
+sudo ./edit-config [[ entry.setup.configuration.file.name ]]
+```
+
+#### Options
+
+[[ entry.setup.configuration.options.description ]]
+
+[% if entry.setup.configuration.options.list %]
+[% if entry.setup.configuration.options.folding.enabled %]
+{% details summary="[[ entry.setup.configuration.options.folding.title ]]" %}
+[% endif %]
+| Name | Description | Default | Required |
+|:----:|-------------|:-------:|:--------:|
+[% for item in entry.setup.configuration.options.list %]
+| [[ item.name ]] | [[ item.description ]] | [[ item.default ]] | [[ item.required ]] |
+[% endfor %]
+[% if entry.setup.configuration.options.folding.enabled %]
+{% /details %}
+[% endif %]
+[% endif %]
```

Comment:
In case it was missed. We need to check for [option.detailed_description](https://github.com/netdata/netdata/blob/1288256c1d68dba448f59d5954a3ce4d74887f04/collectors/metadata/schemas/single-module.json#L322-L324). It should be added after the table.

For example, see [openvpn/per_user_stats](https://github.com/netdata/go.d.plugin/blob/373fe187c9cbcd33dd03d46c319ff8ce56dc1bd9/modules/openvpn/README.md?plain=1#L88-L122)